rollup/da_syncer/da/commitV2.go

```diff
@@ -0,0 +1,38 @@
+package da
+
+import (
+	"context"
+
+	"github.com/scroll-tech/go-ethereum/ethdb"
+	"github.com/scroll-tech/go-ethereum/rollup/da_syncer/blob_client"
+	"github.com/scroll-tech/go-ethereum/rollup/rollup_sync_service"
+
+	"github.com/scroll-tech/go-ethereum/core/types"
+)
+
+type CommitBatchDAV2 struct {
+	*CommitBatchDAV1
+}
+
+func NewCommitBatchDAV2(ctx context.Context, db ethdb.Database,
+	l1Client *rollup_sync_service.L1Client,
+	blobClient blob_client.BlobClient,
+	vLog *types.Log,
+	version uint8,
+	batchIndex uint64,
+	parentBatchHeader []byte,
+	chunks [][]byte,
+	skippedL1MessageBitmap []byte,
+) (*CommitBatchDAV2, error) {
+
+	v1, err := NewCommitBatchDAV1(ctx, db, l1Client, blobClient, vLog, version, batchIndex, parentBatchHeader, chunks, skippedL1MessageBitmap)
```

Comment:
We can't use V1 fully here because `codecv1.DecodeTxsFromBlob` differs from `codecv2.DecodeTxsFromBlob`. `codecv2` also applies decompression